src/app/modules/item/pages/item-log-view/item-log-view.component.html

```diff
@@ -1,9 +1,53 @@
 <ng-container *ngIf="state$ | async as state">
-  <p i18n>Log view</p>
-
   <p-progressSpinner *ngIf="state.isFetching" [style]="{width: '3.3333rem', height: '3.3333rem'}"></p-progressSpinner>
 
-  <p *ngIf="state.isError" i18n>Unable to load the recent activity</p>
+  <p *ngIf="state.isError" class="error-message">
+    <i class="fa fa-exclamation-triangle"></i>
+    <span i18n>Unable to load the recent activity</span>
+  </p>
+
+  <ng-container *ngIf="state.isReady">
+      <p-table #table
+               [columns]="state.data.columns"
+               [value]="state.data.rowData">
+
+        <ng-template pTemplate="header" let-rowData let-columns>
+          <tr *ngIf="rowData.length > 0">
+            <ng-container *ngFor="let col of columns">
+              <th>
+                <div class="header-container">
+                  {{ col.header }}
+                </div>
+              </th>
+            </ng-container>
+          </tr>
```

Comment:
As columns are static here, the code (ts+html) should not be "polluted" (IMO) with computed columns.... hum I see the table I asked you to get inspired from do this...  
Ok, let's keep it this way then.